components/submit/submit-resource-form.tsx

```
@@ -0,0 +1,374 @@
+    mutationFn: async (data: FormValues) => createResource(data),
+    onError: (error) => {
+      toast.error("Failed to submit resource. Please try again.");
+    },
+    onSuccess: () => {
+      form.reset();
+      clearThumbnail();
+      toast.success("Resource submitted successfully!");
+    },
+  });
+  const handleThumbnailChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    if (e.target.files && e.target.files[0]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)